src/styles/desktop.css

```diff
@@ -1,3 +1,9 @@
 body {
+	max-width: 900px;
```

Comment:
This is not part of the footer totally. I think this has to be decided with all the team.

Reply:
Sure, I'll remove that line